assessments/README.md

```diff
@@ -78,3 +78,22 @@ project and the community, where the primary content is generated by the
 
 See [security assessment guide](guide) for more details.  To understand how we
 prioritize reviews, see [intake process](./intake-process.md).
+
+### CNCF and TOC process alignment
+
+**Sandbox** The _self-assessment artifact_ is first created while the project
+is in the **sandbox**.  It is the starting point for projects to consider their security health.
+
+**Incubation** If a self-assessment artifact was not created during sandbox,
+it may be created as part of **incubation**.  It is the starting point for 
+projects to consider their security health.  The _joint-review artifact_ 
+must be created during **incubation** (if not before).
+It leverages the information from the self-assessment and assists projects in 
+becoming more security mature and increasing awareness of their security
+features across the community.
+
+**Graduation** When the project undergoes a CNCF security audit,
+the information from the self-assessment and joint-review are incredibly useful
+for auditors.  Both of these artifacts also assist security teams with a
```

Comment:
Instead of qualitative comment here, can we highlight the fact that it sets the context specifying 
* What security concerns that this project address and to what extent
* What are the ones that it doesn't address
* What are the gaps that is in scope and will be addressed 
Having this information makes the process of audit focussed and provides a meaningful outcome for the project and its success.

Reply:
updated